src/frontend/src/lib/components/ui/Badge.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="ts">
+	export let variant: 'default' | 'info' | 'error' | 'warning' | 'success' | 'outline' = 'default';
+	export let styleClass: string | undefined = undefined;
+
+	const variantClassNames = {
+		default: 'border border-light-grey bg-dust/30',
+		info: 'bg-light-blue text-blue-ribbon',
+		error: 'bg-rusty-red/20 text-rusty-red',
+		warning: 'bg-american-orange/20 text-american-orange',
+		success: 'bg-green-crayola/20 text-green-crayola',
+		outline: 'border border-light-grey bg-off-white'
+	};
+</script>
+
+<div
```

Comment:
Shouldn't it be either a `span` or `li`?

Reply:
I'll make it a span